eng/automation/vs-workload.template.props

```diff
@@ -3,6 +3,7 @@
   <PropertyGroup>
     <TargetName>Microsoft.NET.Sdk.Maui.Workload.@VSMAN_VERSION@</TargetName>
     <ManifestBuildVersion>@VS_COMPONENT_VERSION@</ManifestBuildVersion>
+    <EnableSideBySideManifests>true</EnableSideBySideManifests>
```

Comment:
i think i need to remove this change

Reply:
@pjcollins this is just for net8 branches right?

Reply:
Correct, this should only be enabled for builds that shipped alongside .NET 8 Preview 7 or later.